setup.py

```diff
@@ -32,7 +32,6 @@
         'Acquisition',
         'ExtensionClass>=4.1a1',
         'Persistence',
-        'ZODB3',
```

Comment:
Let's add `BTrees` as an explicit dependency, since this product imports from it directly.